google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java

```diff
@@ -699,7 +704,7 @@ public static EntryListOption filter(String filter) {
    *     WriteOption.resource(MonitoredResource.newBuilder("global").build()));
    * }</pre>
    */
-  void write(Iterable<LogEntry> logEntries, WriteOption... options);
+  ApiFuture<Boolean> write(Iterable<LogEntry> logEntries, WriteOption... options);
```

Comment:
This looks like a breaking change so we'd need to do this as part of a 2.0 google-cloud-logging release.